tests/conftest.py

```diff
@@ -7,6 +8,8 @@
 def pytest_addoption(parser):
     parser.addoption("--loky-verbosity", type=int, default=logging.DEBUG,
                      help="log-level: integer, SUBDEBUG(5) - INFO(20)")
+    parser.addoption("--skip-memory", action="store_true",
```

Comment:
I think we should rename those options to `--skip-high-memory` / `@pytest.mark.high_memory`.